t/convenience.t

```diff
@@ -18,20 +22,52 @@ my %caps = %{ $harness->base_caps };
 $caps{remote_server_addr} = '127.0.0.1';
 delete $caps{browser_name};
 
-my $firefox = Selenium::Firefox->new( %caps );
-ok( $firefox->browser_name eq 'firefox', 'convenience firefox is okay' );
-$firefox->quit;
+subtest Driver => sub {
+	my $phantomjs = Selenium::PhantomJS->new( %caps );
+	ok( $phantomjs->browser_name eq 'phantomjs', 'convenience phantomjs is okay' );
+	$phantomjs->quit;
 
-my $chrome = Selenium::Chrome->new( %caps );
-ok( $chrome->browser_name eq 'chrome', 'convenience chrome is okay' );
-$chrome->quit;
+	my $firefox = Selenium::Firefox->new( %caps );
+	ok( $firefox->browser_name eq 'firefox', 'convenience firefox is okay' );
+	$firefox->quit;
 
-SKIP: {
-    skip 'Can only test IE on windows', 1 unless $^O eq 'MSWin32';
+	my $chrome = Selenium::Chrome->new( %caps );
+	ok( $chrome->browser_name eq 'chrome', 'convenience chrome is okay' );
+	$chrome->quit;
+
+	SKIP: {
+		skip 'Can only test IE on windows', 1 unless $^O eq 'MSWin32';
+
+		my $ie = Selenium::InternetExplorer->new( %caps );
+		ok( $ie->browser_name eq 'internet_explorer', 'convenience ie is okay' );
+		$ie->quit;
+	}
+};
+
+subtest TestDriver => sub {
+	my $phantomjs = Test::Selenium::PhantomJS->new( %caps );
+	ok( $phantomjs->browser_name eq 'phantomjs', 'convenience phantomjs is okay' );
```

Comment:
Both `subtest TestDriver` and `subtest Driver` `ok` messages refer to the driver objects (`Test::Selenium::<BROWSER>` or `Selenium::Chrome::<BROWSER>`) as `convenience`. 

Should `subtest TestDriver` refer to `convenience` and `subtest Driver` to something else (like `normal`)?